perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -201,6 +201,8 @@ sub munge_report_new_bodies {
         my $thamesmead = FixMyStreet::Cobrand::Thamesmead->new({ c => $self->{c} });
         $thamesmead->munge_thamesmead_body($bodies);
     }
+
+    $self->{c}->cobrand->call_hook(munge_overlapping_asset_bodies => $bodies) if $self->{c}->cobrand->can('report_is_on_cobrand_asset');
```

Comment:
I think the `if $self->{c}->cobrand->can('report_is_on_cobrand_asset')` here (and in `UKCouncils.pm` below) should be moved into the `munge_overlapping_asset_bodies` method, as it's more of an implementation detail of that method, not something the caller should have to worry about.